TABLE.md

```diff
@@ -62,4 +62,6 @@
 
 <sup>11</sup> `finisher` function is a callback that is called at the end of class creation. It's optional.
 
+<sup>12</sup> `kind: "initializer"` permits to define a `finisher` callback to be used purely for perform a side effect when instantiating a class. This descriptor don't create new members. Do not confuse with the `initializer` property used to set the initial value to a field.
```

Comment:
I don't think we should describe initializers as finishers, as the timing is different. Instead, let's describe them as field-like things that don't define anything.